tests/homework1/test_task02.py

```diff
@@ -0,0 +1,38 @@
+from homework1.task02 import check_fibonacci
+
+
+def test_fib_sequence_case():
+    """Testing that actual Fibonacci sequence give True"""
+    data_to_process = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144,
+                       233, 377, 610, 987, 1597, 2584, 4181, 6765]
+    assert check_fibonacci(data_to_process)
+
+
+def test_not_a_fib_sequence_case():
+    """Testing that not a Fibonacci sequence give False"""
+    data_to_process = [0, 1, 1, 7]
+    assert not check_fibonacci(data_to_process)
+
+
+def test_single_value_from_fib_sequence_case():
+    """Testing that single number from actual Fibonacci sequence give True"""
+    data_to_process = [2584]
+    assert check_fibonacci(data_to_process)
```

Comment:
regarding comment in task file.. that incorrect sequence

Reply:
Fixed